src/Data/Rational/Properties.agda

```diff
@@ -614,6 +614,15 @@ toℚᵘ-isOrderMonomorphism-< = record
 <-asym : Asymmetric _<_
 <-asym (*<* p<q) (*<* q<p) = ℤ.<-asym p<q q<p
 
+<-dense : Dense _<_
+<-dense {p} {q} p<q with m , p<ᵘm , m<ᵘq ← ℚᵘ.<-dense (toℚᵘ-mono-< p<q)
+        = fromℚᵘ m , p<m , m<q
+  where
+  m≃m : m ≃ᵘ toℚᵘ (fromℚᵘ m)
+  m≃m = ℚᵘ.≃-sym (toℚᵘ-fromℚᵘ m)
```

Comment:
Minor but can we have an empty line between proofs.

Reply:
A-ha! not a style recommendation that I had (yet) internalised.